src/routes/+page.svelte

```diff
@@ -217,7 +217,7 @@
 					class="checkbox"
 				/>
 				<label for="isolate-profile"
-					>Use a different <code class="inline-code">.minecraft</code> directory for this version?</label
+					>Use a subfolder inside the <code class="inline-code">.minecraft</code> folder</label
```

Comment:
the profile folder isn't necessarily limited to a subdirectory inside .minecraft

Reply:
```suggestion
					>Use a different folder for this version</label
```
maybe? not too sure though